units/Legion/Defenses/legstarfall.lua

```diff
@@ -136,6 +136,9 @@ return {
 				weapontimer = 14,
 				weapontype = "Cannon",
 				weaponvelocity = 900,
+				--customparams = {
+				--	stockpilelimit = 1,
+				--},
```

Comment:
Stockpile limit was defined but this weapon does not have stockpile configuration

Reply:
Removed unused weapondef